images.go

```diff
@@ -94,7 +95,7 @@ func printImages(ctx context.Context, clicontext *cli.Context, imageList []image
 	}
 
 	for _, img := range imageList {
-		size, err := img.Size(ctx, cs, platforms.DefaultStrict())
+		size, err := unpackedImageSize(ctx, clicontext, client, img)
 		if err != nil {
 			return errors.Wrap(err, "failed to compute image size")
```

Comment:
This should be logrus.Warn, not return err

Reply:
(Multiple images may face this warning depending on snapshotter, so a warning should be printed only once)

Reply:
@AkihiroSuda changes applied